apps/site/pages/docs/getting-started-glossary.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Glossary'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Glossary
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+  This documentation is currently under development.
+</Callout>
+
+
+  ## C 
+
+  ### Content Type
+  The nature of pages created via the [VTEX Headless CMS](/docs/headless-cms-overview). 
+  A Landing Page, a Product Listing Page (PLP), and a Product Detail Page (PDP) are a few examples of different Content Types.
+
+---
+
+## D 
+
+### Design tokens
+Design elements that define the behavior and appearance of components in a store. 
+There are two types: global tokens, which control system-wide interactions, and local tokens, which customize individual components. 
```

Comment:
Should we link to global tokens? https://www.faststore.dev/docs/global-tokens/overview